source/install/config-ssl-http2-nginx.rst

```diff
@@ -322,22 +293,22 @@ Why does Certbot fail the http-01 challenge?
 
 .. code-block:: none
 
-      Requesting a certificate for yourdomain.com
-      Performing the following challenges:
-      http-01 challenge for yourdomain.com
-      Waiting for verification...
-      Challenge failed for domain yourdomain.com
-      http-01 challenge for yourdomain.com
-      Cleaning up challenges
-      Some challenges have failed.
+  Requesting a certificate for yourdomain.com
+  Performing the following challenges:
+  http-01 challenge for yourdomain.com
+  Waiting for verification...
+  Challenge failed for domain yourdomain.com
+  http-01 challenge for yourdomain.com
+  Cleaning up challenges
+  Some challenges have failed.
    
 If you see the above errors this is typically because certbot was not able to access port 80. This can be due to a firewall or other DNS configuration. Ensure that your A/AAAA records are pointing to this server and your ``server_name`` within the NGINX config does not have a redirect.
```

Comment:
```suggestion
If you see the above errors this is typically because Certbot was not able to access port 80. This can be due to a firewall or other DNS configuration. Make sure that your A/AAAA records are pointing to this server and your ``server_name`` within the NGINX config doesn't have a redirect.
```